documentation/public/user-guide/07-semantic-analysis/common/compilation-builder.mts

```diff
@@ -0,0 +1,15 @@
+import { CompilationBuilder, CompilationUnit } from "@nomicfoundation/slang/compilation";
+import { readFile } from "./read-file.mjs";
+import { resolveImport } from "./resolve-import.mjs";
+
+export async function buildCompilationUnit(): Promise<CompilationUnit> {
+  const builder = CompilationBuilder.create({
+    languageVersion: "0.8.22",
```

Comment:
This is one minor inconsistency with the rest of the guides. We need 0.8.22 here because the example has a top level Event definition, which is was not supported in older versions. But in the rest of the examples, we're using 0.8.0. Should we change them to 0.8.22 (or even a newer version)? Or maybe add a comment here explaining why we need this different version? Or just ignore it and move on... :smile:

Reply:
I'll update all for consistency.